roles/phantomjs/tasks/main.yml

```diff
@@ -0,0 +1,91 @@
+---
+- name: Accept the MS TrueType fonts installer EULA
+  command: echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
```

Comment:
The `ttf-mscorefonts-installer` package is interactive, requiring you to read and accept the EULA. This command lets us do a hands-off install.